src/main/js/controls/Gantt/Gantt.js

```diff
@@ -315,6 +319,59 @@ class Gantt extends Construct {
         return this;
     }
 
+    /**
+     * Updates the graph axisData and content.
+     *
+     *  @returns {Gantt} - Gantt instance
+     */
+    reflow() {
```

Comment:
How would consumer use this if they dont have all the data upfront?
There are 2 cases of using Pan functionality: Consumer does 1 DB call and gets all the data, Consumer does DB call when user clicks on Next or Previous. 
We will need to consider both options in our examples.